osu.Game.Rulesets.Osu/Skinning/Legacy/LegacyCursorParticles.cs

```diff
@@ -67,12 +74,26 @@ private void load(ISkinSource skin)
                     Colour = starBreakAdditive,
                     Direction = SpewDirection.None,
                 },
+                relaxHitSpewer = new LegacyRelaxCursorParticleSpewer(texture, 60)
+                {
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    Colour = starBreakAdditive,
+                    Direction = SpewDirection.RelaxHit,
+                }
             };
 
             if (player != null)
                 ((IBindable<bool>)breakSpewer.Active).BindTo(player.IsBreakTime);
+
+            relaxing = isRelaxing();
+
+            if (relaxing)
+                kiaiSpewer.Active.BindValueChanged(_ => { });
         }
 
+        private bool isRelaxing() => player != null && player.Mods.Value.OfType<OsuModRelax>().Any();
```

Comment:
Very rarely do we allow this sort of stuff where a drawable component queries active mods. Maybe fine in this case, not sure.